homotopy-graphics/src/geom.rs

```diff
@@ -98,6 +98,7 @@ pub struct VertData {
     pub position: Vec4,
     pub boundary: [bool; 4],
     pub generator: Generator,
+    pub k: usize,
```

Comment:
Perhaps you could introduce a typealias or new type for this `usize`? What's the maximum value likely to be (could we get away with a `u32` or `u16`)? `k` is also not a very informative name haha

Reply:
Yeah very true. I'll get something better implemented soon.